src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -221,11 +228,10 @@ bool mongoConnect(const char* host, const char* db, const char* rplSet, const ch
 * mongoConnect -
 *
 * Version of the functions that doesn't uses authentication parameters
-*
 */
 bool mongoConnect(const char* host) {
 
-    return mongoConnect(host, "", "", "", "", false);
+    return mongoConnect(host, "", "", "", "", false, 0);
```

Comment:
If the first mongoConnect used default values, this function would become obsolete